addons/point_of_sale/models/pos_session.py

```diff
@@ -23,7 +23,7 @@ def _confirm_orders(self):
             for order in session.order_ids.filtered(lambda o: o.state != 'done'):
                 if order.state not in ('paid', 'invoiced'):
                     raise UserError(_("You cannot confirm all orders of this session, because they have not the 'paid' status"))
-                order.signal_workflow('done')
+                order.action_pos_order_done()
```

Comment:
(I'm not the new reponsible for PoS, just fixed the bug I introduced during the workflow removal.)